source/index.html.md

```diff
@@ -122,6 +139,39 @@ videojs('my-player', {
 });
 ```
 
+<p class="lang-specific objc">
+  To monitor the performance of an AVPlayer, call either <code>monitorAVPlayerViewController:withPlayerName:andConfig:</code> or <code>monitorAVPlayerLayer:withPlayerName:andConfig:</code>, passing a pointer to your AVPlayer container to the SDK. When calling <code>destroyPlayer</code> or <code>videoChangeForPlayer:withConfig:</code> to <a href="#changing-the-video">change the video</a> the same player name used for the monitor call must be used.
```

Comment:
`passing a pointer to your AVPlayer container (either the AVPlayerLayer or AVPlayerViewController)`